src/App.js

```diff
@@ -1,24 +1,18 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```

Comment:
Consider separating third-party imports from the project's ones.
So first imports in a file should be third-party imports like "react".
Then your project common imports that appear in almost all the components - like for example "../../utils/your-util.js".
Then your component related stuff - like "./your-component-helper.js".
And then the css imports.

Every import group should be separated from other import groups with an empty new line.

Don't forget to do this in all other files of the project.

Reply:
```
import React from "react";


import Header from "./components/Header/Header";
import Navbar from "./components/Navbar/Navbar";
import Profile from "./components/Profile/Profile";

import "./App.css";
```

like so?